src/fragments/Chat.tsx

```diff
@@ -43,6 +44,12 @@ const Chat: React.FC = () => {
             )
           );
         }
+      },
```

Comment:
This should happen inside `ChatAppContextProvider`, where the `messages` belongs. The `ChatAppContextProvider` component is generally responsible for maintaining the application state.